src/main/scala/ophan/google/indexing/observatory/model/Site.scala

```diff
@@ -2,12 +2,13 @@ package ophan.google.indexing.observatory.model
 
 import java.net.URI
 
-case class Site(url: String, searchEngineId: String, sitemaps: Set[URI])
+// Note, the datastore ID is only used but a connected app is still necessary for this to work
```

Comment:
I'm not quite sure what this comments means?

Reply:
yeah i need to add more context. I can add it in the docs.

Reply:
Have updated the docs